### ISSUE TITLE
DVBSky S960 finds few channels when scanning on Cubox-I4Pro

### ISSUE THREAD

**Issue by schaal** [OpenELEC/OpenELEC.tv#4140]
After scanning for channels (either using the VNSI OSD or w_scan) only few channels are found and most HD channels are missing. (using OpenElec 5.95.1)

Also tried scanning using OpenElec 5.0.8 on an RPi2 and using a different distro on the Cubox using Linux 4.0.2 which found more channels.

Header of the w_scan output:
```
w_scan -f s -s s19e2 -R 1 -O 0 -E 0 -o 21 
w_scan version 20141122 (compiled for DVB API 5.10)
using settings for 19.2 east Astra 1F/1G/1H/1KR/1L
scan type SATELLITE, channellist 67
output format vdr-2.1
WARNING: could not guess your codepage. Falling back to 'UTF-8'
output charset 'UTF-8', use -C <charset> to override
Info: using DVB adapter auto detection.
	/dev/dvb/adapter0/frontend0 -> SATELLITE "Montage M88DS3103": very good :-))

Using SATELLITE frontend (adapter /dev/dvb/adapter0/frontend0)
-_-_-_-_ Getting frontend capabilities-_-_-_-_ 
Using DVB API 5.10
frontend 'Montage M88DS3103' supports
INVERSION_AUTO
DVB-S
DVB-S2
FREQ (0.95GHz ... 2.15GHz)
SRATE (1.000MSym/s ... 45.000MSym/s)
using LNB "UNIVERSAL"
-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_ 
```

When scanning on the Cubox, a lot of transponders return "no signal":
```
tune to: S2 f = 11494 kHz H SR = 22000  2/3 0,35  8PSK  (1:1:1019) (time: 06:10.553) 
----------no signal----------
tune to: S2 f = 11494 kHz H SR = 22000 AUTO 0,AUTO  8PSK  (1:1:1019) (time: 06:16.016)  (no signal)
----------no signal----------
tune to: S  f = 11509 kHz V SR = 22000  5/6 0,35  QPSK  (1:1:1020) (time: 06:21.456) 
----------no signal----------
tune to: S  f = 11509 kHz V SR = 22000 AUTO 0,AUTO  QPSK  (1:1:1020) (time: 06:26.876)  (no signal)
----------no signal----------
```

Corresponding w_scan output from the RPi2:
```
tune to: S2 f = 11494 kHz H SR = 22000  2/3 0,35  8PSK  (1:1:1019) (time: 06:05.356) 
	service = Das Erste HD (ARD)
	service = arte HD (ARD)
	service = SWR BW HD (ARD)
	service = SWR RP HD (ARD)
        already known: (S2 f = 11720 kHz H SR = 28200  5/6 0,25  8PSK  (3:3:3201)), but not found by pids
        already known: (S2 f = 11739 kHz V SR = 27500  2/3 0,25  8PSK  (3:3:3202)), but not found by pids
        already known: (S2 f = 11778 kHz V SR = 27500 9/10 0,35  QPSK  (3:3:3204)), but not found by pids
        already known: (S2 f = 11798 kHz H SR = 29500  3/4 0,20  8PSK  (3:3:3205)), but not found by pids
        already known: (S  f = 11837 kHz H SR = 27500  5/6 0,35  QPSK  (3:3:3207)), but not found by pids
        already known: (S2 f = 11856 kHz V SR = 27500  2/3 0,25  8PSK  (3:3:3208)), but not found by pids
        already known: (S2 f = 11876 kHz H SR = 29900  3/4 0,25  8PSK  (3:3:3209)), but not found by pids
        already known: (S  f = 11895 kHz V SR = 27500  5/6 0,35  QPSK  (3:3:3210)), but not found by pids
        already known: (S2 f = 11915 kHz H SR = 29900  2/3 0,20  8PSK  (3:3:3211)), but not found by pids
        already known: (S  f = 11973 kHz V SR = 27500  5/6 0,35  QPSK  (3:3:3214)), but not found by pids
        already known: (S2 f = 12032 kHz H SR = 27500 9/10 0,35  QPSK  (3:3:3217)), but not found by pids
        already known: (S  f = 12071 kHz H SR = 27500  3/4 0,35  QPSK  (3:3:3219)), but not found by pids
        already known: (S2 f = 12110 kHz H SR = 27500  3/4 0,20  8PSK  (3:3:3221)), but not found by pids
        already known: (S2 f = 12129 kHz V SR = 27500  2/3 0,25  8PSK  (3:3:3222)), but not found by pids
        already known: (S2 f = 12149 kHz H SR = 27500  5/6 0,25  8PSK  (3:3:3223)), but not found by pids
        already known: (S  f = 12168 kHz V SR = 27500  3/4 0,35  QPSK  (3:3:3224)), but not found by pids
        already known: (S2 f = 12188 kHz H SR = 27500  2/3 0,25  8PSK  (3:3:3225)), but not found by pids
        already known: (S2 f = 12207 kHz V SR = 27500  5/6 0,20  8PSK  (3:3:3226)), but not found by pids
        already known: (S2 f = 12305 kHz H SR = 27500  5/6 0,25  8PSK  (3:3:3231)), but not found by pids
        already known: (S2 f = 12363 kHz V SR = 29500  3/4 0,20  8PSK  (3:3:3234)), but not found by pids
        already known: (S2 f = 12383 kHz H SR = 30000  8/9 0,25  8PSK  (3:3:3235)), but not found by pids
        already known: (S2 f = 12402 kHz V SR = 30000  3/4 0,20  8PSK  (3:3:3236)), but not found by pids
```

**Comment by stefansaraev** [comment]
cubox project using an non maintained, non-mainline dvbsky driver, nothing we can do.

**Comment by schaal** [comment]
Fixed by #4141
